streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/task/CheckpointProcessor.java

```diff
@@ -131,6 +131,18 @@ private void process(Application application, @Nonnull CheckPoints.CheckPoint ch
     }
   }
 
+  // issue: https://github.com/apache/incubator-streampark/issues/3749
```

Comment:
We should avoid this meaningless comment since we should do that in commit message and pr.

Reply:
> We should avoid this meaningless comment since we should do that in commit message and pr.

Thanks, updated